config.json

```diff
@@ -1,63 +1,83 @@
 {
-  "track_id": "gnu-apl",
-  "language": "GNU APL",
   "active": false,
-  "test_pattern": "\\.tc$",
+  "deprecated": [],
   "exercises": [
     {
-      "uuid": "81121f0c-1218-4d88-bb6c-fd9df3bc0b8d",
-      "slug": "hello-world",
       "core": true,
       "difficulty": 1,
-      "topics": ["setup"]
+      "slug": "hello-world",
+      "topics": [
+        "setup"
+      ],
+      "uuid": "81121f0c-1218-4d88-bb6c-fd9df3bc0b8d"
     },
     {
-      "uuid": "9a89822e-0d61-f980-49fa-c273c0a0e0869b9fc7f",
-      "slug": "leap",
       "core": true,
       "difficulty": 1,
-      "topics": ["conditions", "dates"]
+      "slug": "leap",
+      "topics": [
+        "conditions",
+        "dates"
+      ],
+      "uuid": "9a89822e-0d61-f980-49fa-c273c0a0e0869b9fc7f"
     },
     {
-      "uuid": "bc2c01ff-0e38-b280-5047-2014e184502c67074c0",
-      "slug": "hamming",
       "core": true,
       "difficulty": 1,
-      "topics": ["strings", "arrays", "differences"]
+      "slug": "hamming",
+      "topics": [
+        "arrays",
+        "differences",
+        "strings"
+      ],
+      "uuid": "bc2c01ff-0e38-b280-5047-2014e184502c67074c0"
     },
     {
-      "uuid": "05764380-02d5-2a80-0741-3baa1e10b36b61333ec",
-      "slug": "rna-transcription",
       "core": true,
       "difficulty": 1,
-      "topics": ["strings", "arrays", "substitution", "error handling"]
+      "slug": "rna-transcription",
+      "topics": [
+        "arrays",
+        "error_handling",
```

Comment:
I thought topic names were meant to be human-readable, so why the underscores? Am I wrong about the human-readability?